src/App.js

```diff
@@ -1,25 +1,83 @@
-import logo from './logo.svg';
-import './App.css';
+/** @format */
+import "./App.css";
+import axios from "axios";
+import { Routes, Route, Link } from "react-router-dom";
+import HomePage from "./components/HomePage";
+import NavBar from "./components/NavBar";
+import Header from "./components/Header";
+import { useEffect, useState } from "react";
+import BeersList from "./components/BeersList";
+import BeerDetails from "./components/BeerDetails";
+import CreateBeer from "./components/CreateBeer";
+import Search from "./components/Search";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+	const [beers, setBeers] = useState([]);
+	const [search, setSearch] = useState("");
+
+	useEffect(() => {
+		getBeersFromApi();
```

Comment:
Because you're fetching the beers in the App.js you need to pass the function also to the CreateBeer component, so you can update the info that we got on the view